### PR TITLE
RHDEVDOCS 5244 Tekton Results documentation

### PR DESCRIPTION
**Pull request by mramendi**
Version(s):

needs cp to pipelines-docs-1.12

Issue:

RHDEVDOCS 5244

Link to docs preview:

QE review:
- [x] QE has approved this change.

Additional information:

This content was fully reviewed by Dev, QE, and a peer reviewer in https://github.com/openshift/openshift-docs/pull/63780 , however that PR will not go ahead as the Pipelines documentation is no longer merged into main. Instead this PR needs a merge to pipelines-docs and cp into pipelines-docs-4.12

### CODE REVIEW

**Comment by mramendi** [comment]
Last moment engineering hold, please do not merge

**Comment by mramendi** [comment]
(engineering gave a last moment change which consisted purely of removing two lines from YAML examples, so should not need a redoing of peer reviews. The change is now applied)

File: modules/op-installing-results.adoc [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+    prometheus_port: 9090
+----
+
+. Add configuration for the storage for logging information to this file:
+** If you configured a persistent volume claim (PVC), add the following line to provide the name of the PVC:
++
+[source,yaml]
+----
+    logging_pvc_name: tekton-logs
+----
```
Comment (mburke5678) [inline modules/op-installing-results.adoc:51]: Should add the following in this and the following code blocks, per the [Doc Guidelines](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#required-fields):

```
apiVersion: operator.tekton.dev/v1alpha1
kind: TektonResult
metadata:
  name: result
spec:
# ...
```

File: modules/op-results-concepts.adoc [hunk 2]
```diff
@@ -0,0 +1,172 @@
...
+A _result_ can contain one or several records. A record is always a part of exactly one result.
+
+A result corresponds to a pipeline run, and includes the records for the `PipelineRun` CR itself and for all the `TaskRun` CRs that were started as a part of the pipeline run.
+
+If a task run was started directly, without the use of a pipeline run, a result is created for this task run. This result contains the record for the same task run.
+
+Each result has a name that includes the namespace in which the `PipelineRun` or `TaskRun` CR was created and the UUID of the CR.  The format for the result name is `<namespace_name>/results/<parent_run_uuid>`. In this format, `<parent_run_uuid>` is the UUUD of a pipeline run or else of a task run that was started directly.
+
+.Example result name
+----
```
Comment (mburke5678) [inline modules/op-results-concepts.adoc:22]: Needs a [source, ****] tag. Maybe [source,terminal]?

File: modules/op-installing-results.adoc (hunk 1 above)
Comment (mramendi) [reply modules/op-installing-results.adoc:51]: @mburke5678 in this part of the procedure, I do not provide a YAML example, to which the guidance applies, Instead, I am explicitly instructing the user to _add_ lines to a file that the user just created in the previous step. If I put in the full header, the user could possibly add this header again, and this breaks the file.

**Pull request merged**